spec-tests/src/test/java/org/candlepin/spec/OwnerResourceTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *  Copyright (c) 2009 - ${YEAR} Red Hat, Inc.
+ *
+ *  This software is licensed to you under the GNU General Public License,
+ *  version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ *  implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ *  FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ *  along with this software; if not, see
+ *  http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ *  Red Hat trademarks are not licensed under GPLv2. No permission is
+ *  granted to use or replicate Red Hat trademarks that are incorporated
+ *  in this software or its documentation.
+ */
+package org.candlepin.spec;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.candlepin.ApiException;
+import org.candlepin.dto.api.v1.NestedOwnerDTO;
+import org.candlepin.dto.api.v1.OwnerDTO;
+import org.candlepin.resource.OwnerApi;
+import org.candlepin.spec.bootstrap.client.ApiClientFactory;
+import org.candlepin.spec.bootstrap.client.ClientBuilders;
+import org.candlepin.spec.bootstrap.client.Config;
+import org.candlepin.spec.bootstrap.client.OwnerClient;
+import org.candlepin.spec.bootstrap.data.Owners;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@SpecTest
+public class OwnerResourceTest {
+
+    private ApiClientFactory apiClient = ClientBuilders.instance();
+    private OwnerClient api;
+
+    @BeforeEach
+    void setUp() {
+        this.api = new OwnerClient(apiClient.createInstance());
```

Comment:
This kind of setup is going to be everywhere, and several spec tests use several APIs. We may end up in a situation where something like the DatabaseTestFixture ends up being a common parent class.